Q1. WAP to accept integer input from user. 

In [2]:
n=int(input())
print(n)

5


Q2. Write a python program to take input from user age then increase his/her age by 10 years. 
Sample Input : enter you age : 12 
Sample Output : 22 

In [ ]:
n=int(input('age'))
y=n+10
print(y)



22


In [4]:
print(int(input())+10)

22


In [6]:
n=eval(input())
print(n+10)


22


Q3.  Take input user name ,  age  , height  then print output as “my name is (your name),my age is (you age), and height is (your age)” . 
sample Input: Enter you name : alice 
Enter your age : 10 
Enter your height : 3.5 
Sample Output: 
My name is alice, my age is 10 and height is 3.5 

In [ ]:
name=input()
age=int(input())
height=float(input())
# print('my name is %s \nmy age is %d \nmy height is %f'%(name,age,height)) #using format specifiyer
# print('name: %s'%name, 'age:%d'%age, 'height:%f'%height)
# print('my name is %s, my age is %d, my height is %f'%(name,age,height))  
# print("my name is {} \nmy age is {} \nmy height is {:.2f}" .format(name,age,height))  #using foramt() function
print(f'my name is {name} \nmy age is {age} \nmy height is {height:.2f}') # using f'formating

my name is saquib 
my age is 23 
my height is 234.00


In [34]:
name,age,height=input('name'),int(input('age')),float(input('height'))
print('my name is %s \nmy age is %d \nmy height is %f' %(name,age,height))

my name is saquib 
my age is 23 
my height is 234.000000


Q4. How do you input a floating-point number from the user and store it in a variable called price? 

In [ ]:
price=float(input())
print(price)


Q5. Write a Python program that asks the user for two numbers and then prints their sum 

Sample Input : Enter 1st numbers : 3 
               Enter 2nd number :4 
Sampel Output : 7 

In [43]:
a=eval(input())
b=eval(input())
x=a+b
print(x)



saquibmahfuz


Q6.Write a Python program that asks the user for their age and then prints a message stating their age. 

In [46]:
n=int(input('age'))
print('%d years old'%n)

12 years old


Q7.Write a Python program that asks the user for the radius of a circle and then prints the area of the circle. 

In [48]:
r=eval(input())
pi=3.14
area=2*pi*r**2
print(area)

100.48


Q8. How do you take a string input from the user and convert it to an integer? 

Sample input : ‘234’ 
Sample output : 234 

In [ ]:
n=input()
print(int(n))


<class 'str'>
123
<class 'int'>


Q9. Write a Python program that asks the user for their height in feet and converts it to meters (1 foot = 0.3048 meters).

In [61]:
h=eval(input('enter your hight in feet'))
m=h*0.3048
print(m)

1.76784


Q10. Write a Python program that takes three inputs from the user: two integers and one floating-point number. Your task is to print these three values in the following format: 
Integer1  Integer2  Float_Number 

Sample Input: 
Enter the first integer: 5 
Enter the second integer: 10 
Enter the floating-point number: 3.14 
Sample output: 
5 10 3.14 

In [62]:
a=int(input())
b=int(input())
c=float(input())
print(a, b, c)

1 2 4.0
